### ISSUE TITLE
[Question] how to change eol of auto generated d.ts

### ISSUE THREAD

**Issue by kingyue737** [antfu/unplugin-auto-import#197]
In my project, eol is set as 'crlf' in rule of prettier. However, each time I launch vite, eol in `auto-import.d.ts` is replaced with 'lf'. Is there a recommended way to solve this? As this file is the only one in `./src` folder which is generated by a package, it seems not elegant to create prettier and eslint ignore file just for it. Maybe there is another way?

**Comment by antfu** [comment]
Maybe we could generate the disable comments for prettier and eslint out-of-box. WDYT?

**Comment by kingyue737** [comment]
> Maybe we could generate the disable comments for prettier and eslint out-of-box. WDYT?

Good Idea!

**Comment by azaleta** [comment]
> Maybe we could generate the disable comments for prettier and eslint out-of-box. WDYT?

eslint: Easy Mode
prettier: Hell Mode

according to [#5287](https://github.com/prettier/prettier/issues/5287)
prettier don't support block disable in TS/JS
so the dts file will be something like
``` typescript
  // prettier-ignore
  const $$: typeof import('vue/macros')['$$']
  // prettier-ignore
  const $: typeof import('vue/macros')['$']
  // prettier-ignore
  const $computed: typeof import('vue/macros')['$computed']
```
this is really annoying

**Comment by JounQin** [comment]
Why you care about auto-generated files? They should be listed in `.prettierignore` IMO.

**Comment by kingyue737** [comment]
closed by 13b6102
